.travis.yml

```diff
@@ -144,7 +91,7 @@ matrix:
     # g++ build (default builds all use clang++)
     - os: linux
       env: BUILDTYPE=debug CXX="g++-6" CC="gcc-6"
-      node_js: 4
+      node_js: 10
```

Comment:
I'm wondering if we should upgrade GCC version as well ? /cc @springmeyer

Reply:
No strong preference on gcc. This was added to get access to the `-Weffc++` warnings mainly.

Reply:
@artemp feel free to upgrade the GCC version in this PR if you'd like.
I don't have much experience there, so I'd like to leave it for you to use the best version :)